estudios_socioeconomicos/test_models.py

```diff
@@ -0,0 +1,214 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+
+from familias.models import Familia
+from perfiles_usuario.models import Capturista
+from .models import Estudio, Seccion, Pregunta, OpcionRespuesta, Respuesta
+
+
+class EstudioTestCase(TestCase):
+    """ Suite to Test things related to the Estudio Model.
+
+    Attributes:
+    -----------
+    user : User
+        A mock user to use as capturista.
+    capturista : Capturista
+        A mock capturista to use as the one who filled the study.
+    familia : Familia
+        The family of which the study is about.
+    estudio : Estudio
+        The actual study.
+    """
+
+    def setUp(self):
+        """ Setup attributes
+        """
```

Comment:
Single lines also need the \n. https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections